src/common/low_precision_transformations/tests/reduce_sum_transformation.cpp

```diff
@@ -336,4 +337,68 @@ INSTANTIATE_TEST_SUITE_P(
         ::testing::ValuesIn(reduceSumTransformationTestValues)),
     ReduceSumTransformation::getTestCaseName);
 } // namespace testValues2
+
+namespace testValues3 {
+    const std::vector<ov::PartialShape> inputShapes = {
+        {4, 3, 16, 16}
+    };
+
+    const std::vector<ReduceTransformationTestValues> reduceSumTransformationTestValues = {
+        {
+            LayerTransformation::createParamsU8I8(),
+            {0},
+            true,
+            {
+                ov::element::u8,
+                {
+                    {ov::element::f32},
+                    {{2.f, 4.f, 6.f}, ov::element::f32, {1, 3, 1, 1}, false, 1ul, ov::element::u8, true},
```

Comment:
Can I ask you to make one of the zero point values big enough to cover the case when zero point after transformation will be bigger than u8 max value?
```suggestion
                    {{2.f, 4.f, 128.f}, ov::element::f32, {1, 3, 1, 1}, false, 1ul, ov::element::u8, true},
```

Reply:
done, thanks